src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -1094,10 +1099,16 @@ namespace ReportSizingManager {
 						NominalCapacityDes = DesMassFlow * DataWaterCoilSizHeatDeltaT * Cp * rho;
 						// if coil is part of a zonal unit, calc coil load to get hot water flow rate
 					} else {
+						if ( ZoneEqSizing( CurZoneEqNum ).SystemAirFlow ) {
+							DesMassFlow = ZoneEqSizing( CurZoneEqNum ).AirVolFlow * StdRhoAir;
+						} else if ( ZoneEqSizing( CurZoneEqNum ).HeatingAirFlow ) {
```

Comment:
The logic flow here seems a little suspect.  I would expect an if - else-if block to be such that only one would be satisfied at a given time, but here both ZoneEqSizing( CurZoneEqNum ).SystemAirFlow and ZoneEqSizing( CurZoneEqNum ).HeatingAirFlow are often true at the same time, like it is in the unit test now.   So it will flow to the if and skip the else-if even though it is also satisfied.   That may be what you want but it seems hard to follow.